### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by MadhukarYadav**

### CODE REVIEW

File: src/FormComp.js [hunk 1]
```diff
@@ -103,7 +110,10 @@ class FormComp extends Component {
               <NumberInput name='oxygenLevel' value={this.state.oxygenLevel} step={10}
                           onChange={(e) => this.handleUserInput(e)} />
             </FormField>
-            <Button onClick={this.SubmitForm} primary={true} label='Check' style={{margin: '10px 0'}} />
```
Comment (MadhukarYadav) [inline src/FormComp.js:106]: Nice Change, can you also change the main div tag to <Box align='center'>

**Comment by MadhukarYadav** [comment]
test

**Comment by MadhukarYadav** [comment]
test merge

**Review by Madhukar-Yadav — APPROVED** [review]
Nice Work.

**Comment by Madhukar-Yadav** [comment]
UI Fixes merged to the master branch.

**Pull request merged**